mcs/class/System.Drawing/Test/System.Drawing/SystemIconsTest.cs

```diff
@@ -62,14 +62,13 @@ public void Dispose_Indirect ()
 		}
 
 		[Test]
-		[ExpectedException (typeof (ObjectDisposedException))]
 		public void Clone_Dispose ()
 		{
 			// Clones of SystemIcons icon's can be disposed
 			Icon app = SystemIcons.Application;
 			Icon clone = (Icon) app.Clone ();
 			clone.Dispose ();
-			Assert.IsNotNull (clone.ToBitmap ());
+            Assert.Throws<ObjectDisposedException>(() => Assert.IsNotNull (clone.ToBitmap ()));
```

Comment:
Indentation. I'll stop commenting on this, there are a lot more cases in this PR. Please fix them all :)